src/app/_components/DontHaveAnAccount/DontHaveAnAccount.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box, Typography } from "@mui/material";
+
+const DontHaveAnAccount = (): JSX.Element => {
+  return (
+    <Box display="flex" flexDirection="row" gap={1}>
+      <Typography id="dont-have-an-account" color={"#00000066"}>
+        Don’t have an account?
+      </Typography>
+      <Typography id="sign-up-link" color={"#258BB6"}>
```

Comment:
I think this color will be our primary color, until we have a `mui` `theme` setup, you should leave a `// TODO: xxxxx` here to update this color with `theme` later.

Reply:
Resolved